docs/usage.md

```diff
@@ -16,7 +18,28 @@ Click the icon to open the The **FortiAI** bot dialog to begin your interaction
 To manually clear a conversation in the bot, click the **Clear Conversation** button.
 
 >[!Note]
->Clearing a conversation deletes the thread and hence generative AIs, like OpenAI, may lose context.
+>Clearing a conversation deletes the conversation and starts a new thread. Hence generative AIs, like OpenAI, may lose context.
```

Comment:
Now it does not delete a thread @cs-abhishek-shukla, we only clear the conversation